src/main/java/org/altbeacon/beacon/service/BeaconService.java

```diff
@@ -329,6 +339,7 @@ private PendingIntent getRestartIntent() {
     /**
      * methods for clients
      */
+    @MainThread
```

Comment:
Is this annotation intended to be used this way?  The [docs](https://developer.android.com/reference/android/support/annotation/MainThread.html) say it is supposed to indicate that the method should only be called on the main thread, and I don't see why that would be true in this case.  It seems to be intended for methods that manipulate the UI.  In this case, I don't think there is anything wrong from calling on a worker thread.  Perhaps a comment would be a better way to indicate that it is not necessarily run on a worker thread.